frappe/utils/password.py

```diff
@@ -48,13 +48,15 @@ def get_decrypted_password(doctype, name, fieldname='password', raise_exception=
 	elif raise_exception:
 		frappe.throw(_('Password not found'), frappe.AuthenticationError)
 
+
```

Comment:
We follow the convention of leaving 2 line spaces after a class 1 line between functions for visual separation of a class method and other functions.